docs/logger-mdc-instrumentation.md

```diff
@@ -19,11 +19,12 @@ event's MDC copy:
   (same as `Span.current().getSpanContext().getTraceFlags().asHex()`).
 
 Those three pieces of information can be included in log statements produced by the logging library
-by specifying them in the pattern/format. Example for Spring Boot configuration (which uses
-logback):
+by specifying them in the pattern/format. 
 
+Example for Spring Boot configuration, which uses
+logback and the [last tip of this section](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.logging.custom-log-configuration):
```

Comment:
I had not seen this tip before, so it took me a while to realize that `level` was not a mistake 😄

```suggestion
Tip: for Spring Boot configuration which uses logback, you can add MDC to log lines by overriding only the logging.pattern.level:
```